### ISSUE TITLE
Read up on the demands for the website to be as accessable as possible

### ISSUE THREAD

**Issue by aslinedvinsson** [aslinedvinsson/Axisofaccesshack#18]

**Comment by AshLaw96** [comment]
### Accessibility standards

**Perceivable:** Visitors must be able to perceive or understand and be aware of the content and information on your site. Like, having access to alt text.
**Operable:** Visitors must be able to use every part of your site without disruption. A well mapped and organised navigation menu can achieve this.
**Understandable:** All content on your site, whether written or otherwise presented, should be easy to understand. Clear, concise language and clean, uncluttered pages can be used to communicate information.
**Robust:** The underpinnings of your site, like its HTML code, should be easily read and interpreted by all visitors, including assistive technology like screen readers. Another example of this is optimising your website for various devices like phones and tablets.

Got some good info [here](https://www.dreamhost.com/blog/make-your-website-accessible/)
